corehq/apps/ota/views.py

```diff
@@ -154,7 +154,7 @@ def claim(request, domain):
     return HttpResponse(status=200)
 
 
-def get_restore_params(request):
+def get_restore_params(request, domain):
```

Comment:
Do you know if `request.domain` is set? When I saw this, my first thought was wondering whether we could eliminate having to pass in this parameter, but even if you could it's a minor thing

Reply:
i'm not 100% sure if `request.domain` is actually set here (and looks like it isn't for at least one of the two uses), that's why I'm passing in domain explicitly